openpathsampling/experimental/simstore/test_storable_function.py

```diff
@@ -223,6 +223,24 @@ def get_expected(uuid):
 
         self.func = StorableFunction(get_expected)
 
+    @pytest.mark.parametrize('min_max', [(None, None), (None, 10), (0, 10)])
+    def test_check_periodic(self, min_max):
+        period_min, period_max = min_max
+        n_nones = Counter(min_max)[None]
+        expected = {2: False, 0: True, 1: 'error'}[n_nones]
+        check_period = StorableFunction._check_period
+        if expected == 'error':
+            with pytest.raises(RuntimeError):
+                check_period(period_min, period_max)
```

Comment:
maybe make sure that the expected error is raised?
Something like:
```suggestion
            with pytest.raises(RuntimeError, match='period'):
                check_period(period_min, period_max)
```

Reply:
Agree, although I'm going to change this to a `ValueError`. Types `None` or numeric are allowed for `period_min` and `period_max`, but if one is `None` and the other is numeric, the values are in conflict.

Reply:
> Agree, although I'm going to change this to a ValueError.

+1, I do agree that is a better error type